src/backgroundQueue/sinks/memberBundleRequestTrigger.sink.spec.ts

```diff
@@ -33,14 +61,100 @@ describe('triggerBundleRequest', () => {
     await postEvent(triggerEvent, server);
 
     const publishedEvents = getEvents();
-    expect(publishedEvents).toHaveLength(1);
+    expect(publishedEvents).toHaveLength(2);
     expect(publishedEvents).toContainEqual(
       expect.objectContaining<Partial<CloudEvent<MemberBundleRequestPayload>>>({
-        id: 'the public key id',
+        id: MEMBER_PUBLIC_KEY_MONGO_ID,
         source: 'https://veraid.net/authority/bundle-request-trigger',
         type: BUNDLE_REQUEST_TYPE,
-        data: { awalaPda: 'PDA, base64-encoded', publicKeyId: 'the public key id' },
+
+        data: {
+          awalaPda: AWALA_PDA,
+          publicKeyId: MEMBER_PUBLIC_KEY_MONGO_ID,
+        },
       }),
     );
+    expect(publishedEvents).toContainEqual(
+      expect.objectContaining<Partial<CloudEvent<MemberBundleRequestPayload>>>({
+        id: mongoId,
+        source: 'https://veraid.net/authority/bundle-request-trigger',
+        type: BUNDLE_REQUEST_TYPE,
+
+        data: {
+          awalaPda: AWALA_PDA,
+          publicKeyId: mongoId,
+        },
+      }),
+    );
+  });
+
+  test('Database entry should be deleted after event is emitted', async () => {
+    const requestData = {
+      publicKeyId: MEMBER_PUBLIC_KEY_MONGO_ID,
+      memberBundleStartDate: new Date(),
+      signature: SIGNATURE,
+      awalaPda: Buffer.from(AWALA_PDA, 'base64'),
+    };
+
+    const memberBundleRequest = await memberBundleRequestModel.create(requestData);
+
+    const triggerEvent = new CloudEvent<MemberBundleRequestTriggerPayload>({
+      id: CE_ID,
+      source: CE_SOURCE,
+      type: BUNDLE_REQUEST_TRIGGER_TYPE,
+    });
+
+    await postEvent(triggerEvent, server);
+
+    const publishedEvents = getEvents();
+    expect(publishedEvents).toHaveLength(1);
+    const memberBundleRequestCheck = await memberBundleRequestModel.findById(
+      memberBundleRequest._id,
+    );
+    expect(memberBundleRequestCheck).toBeNull();
+  });
+
+  test.each([
+    ['Future', BUNDLE_REQUEST_DATE_RANGE],
+    ['Past', -BUNDLE_REQUEST_DATE_RANGE],
+  ])('%s bundle start date should be emitted', async (_type: string, range: number) => {
+    const requestData = {
+      publicKeyId: MEMBER_PUBLIC_KEY_MONGO_ID,
+      memberBundleStartDate: addDays(new Date(), range),
+      signature: SIGNATURE,
+      awalaPda: Buffer.from(AWALA_PDA, 'base64'),
+    };
+    await memberBundleRequestModel.create(requestData);
+    const triggerEvent = new CloudEvent<MemberBundleRequestTriggerPayload>({
+      id: CE_ID,
+      source: CE_SOURCE,
+      type: BUNDLE_REQUEST_TRIGGER_TYPE,
+    });
+
+    await postEvent(triggerEvent, server);
+
+    const publishedEvents = getEvents();
+    expect(publishedEvents).toHaveLength(1);
+  });
+
+  test('Bundle start date more then 3 days into the future should not be sent', async () => {
+    const data = {
+      publicKeyId: MEMBER_PUBLIC_KEY_MONGO_ID,
+      memberBundleStartDate: addSeconds(addDays(new Date(), BUNDLE_REQUEST_DATE_RANGE), 1),
```

Comment:
A 1-second difference is too small, so this is very likely to be a flaky test. I've been bitten by this too many times on CI, where performance is highly unpredictable 😬 

Let's increase to something like 5 or 10 seconds, which is still a negligible number considering where's dealing with days.